networkit/cpp/centrality/DynBetweennessOneNode.cpp

```diff
@@ -91,7 +91,7 @@ void DynBetweennessOneNode::update(GraphEvent event) {
 		// queue with all visited nodes
 		// if u has a new shortest path going through v, it updates the distance of u
 		// and inserts u in the priority queue (or updates its priority, if already in Q)
-		auto updateQueue = [&](node u, edgeweight priority) {
+		auto updateQueue = [&](node u, edgeweight) {
```

Comment:
`updateQueue` is not passed to any function. Hence, removing the two places of invocation to not pass the second parameter seems better.